command/bridge/exit/exit.go

```diff
@@ -38,33 +38,58 @@ const (
 )
 
 type exitParams struct {
-	txnSenderKey      string
+	accountDir        string
+	accountConfig     string
 	exitHelperAddrRaw string
 	exitID            uint64
 	epochNumber       uint64
 	checkpointBlock   uint64
 	rootJSONRPCAddr   string
 	childJSONRPCAddr  string
+	isTestMode        bool
+}
+
+// validateFlags validates input values
+func (ep *exitParams) validateFlags() error {
+	if !ep.isTestMode {
+		if err := sidechain.ValidateSecretFlags(ep.accountDir, ep.accountConfig); err != nil {
+			return err
+		}
+	} else {
+		if ep.accountDir != "" || ep.accountConfig != "" {
+			return helper.ErrTestModeSecrets
+		}
+	}
+
+	return nil
```

Comment:
Looks like this code is duplicated with https://github.com/0xPolygon/polygon-edge/pull/1264/files#diff-eba234dff888a19051599dcc6f8f65323584f8674c04d66d8c2fbeda353d4a6cR26

Reply:
Fixed https://github.com/0xPolygon/polygon-edge/pull/1264/commits/36a90601fa0b99c584194decca1ec6bc5da8dfe0